sdk/storagepool/Azure.ResourceManager.StoragePool/tests/Scenario/DiskPoolTests.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+using ResourceGroup = Azure.ResourceManager.Resources.ResourceGroup;
+using DiskPoolSku = Azure.ResourceManager.StoragePool.Models.Sku;
+using Azure.ResourceManager.StoragePool.Models;
+using Azure.ResourceManager.Resources.Models;
+using Azure.Core;
+
+namespace Azure.ResourceManager.StoragePool.Tests
+{
+    [TestFixture(true)]
+    public class DiskPoolTests : DiskPoolTestBase
+    {
+        private string SubnetResourceId;
+        private string ManagedDiskId;
+        protected ResourceGroup _resourceGroup;
+
+        public DiskPoolTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public async Task Setup()
+        {
+            _resourceGroup = await CreateResourceGroupAsync();
+            SubnetResourceId = "/subscriptions/bc675d74-3b9f-48da-866a-a10149531391/resourceGroups/synthetics-permanent-canadacentral/providers/Microsoft.Network/virtualNetworks/synthetics-vnet/subnets/synthetics-subnet";
+            ManagedDiskId = "/subscriptions/bc675d74-3b9f-48da-866a-a10149531391/resourceGroups/synthetics-permanent-canadacentral/providers/Microsoft.Compute/disks/synthetics-test-disk";
+        }
+
+        [Test]
+        [RecordedTest]
+        public async Task DiskPoolCrudOperations()
```

Comment:
Please use `Assert` to make sure the test result is the same as expected, you can take this as an [example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Azure.ResourceManager.Compute/tests/Scenario/AvailabilitySetOperationsTests.cs).